ansible-ipi-install/roles/installer/defaults/main.yml

```diff
@@ -1,2 +1,4 @@
 ---
 # defaults file for installer
+increase_bootstrap_timeout: 0
+increase_install_timeout: 0
```

Comment:
No need for default values as I think the goal will be to add new tasks that check if these timeouts are defined and greater than 0

Reply:
What is the value of a check task vs. the defaults? That just seems like overhead to me, but maybe I'm missing something.

Reply:
The "+1" `set_fact`s below leverage these defaults so that we end up with a minimum of `1` for the `wait-for` loops. That's not to say there isn't a better way to do this.